src/NexusMods.App.GarbageCollection/Structs/Hash.cs

```diff
@@ -0,0 +1,5 @@
+using TransparentValueObjects;
+namespace NexusMods.App.GarbageCollection.Structs;
+
+[ValueObject<ulong>]
+public readonly partial struct Hash;
```

Comment:
Why do we need another `Hash` type?

Reply:
This was an accident and has been changed. I accidentally put this PR up before I was done with it.
I was typing the PR text, and accidentally hit `Enter` instead of `Shift+Enter`.

It's a leftover from when I was experimenting on an empty project while running the Nx stress tests in the background.